### PR TITLE
Adding SingleMultiRainbow variant and an option to reverse hands.

### PR DESCRIPTION
**Pull request by jeep**

### CODE REVIEW

File: public/js/ui.js [hunk 1]
```diff
@@ -1232,7 +1232,7 @@ function HanabiUI(lobby, gameID) {
         Kinetic.Group.call(this, config);
 
         this.align = (config.align || 'left');
-        this.reverse = (config.reverse || false);
+        this.reverse = ( lobby.reverseHands ? (config.reverse || false) : !config.reverse);
```
Comment (rogers-j) [inline public/js/ui.js]: You should check the lobby setting when `config.reverse` gets its value, no?

File: public/js/constants.js [hunk 2]
```diff
@@ -938,6 +952,7 @@
         exports.VARIANT.AMBIGUOUS,
         exports.VARIANT.BLUERED,
         exports.VARIANT.ACIDTRIP,
+	exports.VARIANT.MULTISINGLERAINBOW,
```
Comment (rogers-j) [inline public/js/constants.js]: Please make separate commits for separate changes.

File: public/js/dev/game/init.js [hunk 3]
```diff
@@ -345,7 +345,7 @@ function drawPlayers() {
             height: constants.HAND_POS[nump][j].h * globals.ui.h,
             rotation: constants.HAND_POS[nump][j].rot,
             align: 'center',
-            reverse: j === 0,
+            reverse: (lobby.reverseHand ? j != 0 : j === 0),
```
Comment (rogers-j) [inline public/js/dev/game/init.js]: This isn't the same 'branch' as the other code. I suggest not modifying it at the same time as the other branch.

File: public/js/constants.js (hunk 2 above)
Comment (jeep) [reply public/js/constants.js]: Yeah, Zamiel asked me to just submit it despite it not being tested or reviewed. He said he would fix it up.

File: public/js/dev/game/init.js (hunk 3 above)
Comment (rogers-j) [inline public/js/dev/game/init.js]: There's another bit of code we need to reverse the gradient
